src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,28 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  const { id, name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo" key={id}>
```

Comment:
remove key here ( best way )
```suggestion
    <div className="CommentInfo">
```